app/assets/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -199,12 +206,14 @@ class DatasetInfoTabView extends React.PureComponent<DatasetInfoTabProps> {
       <div>
         <p>{annotationTypeLabel}</p>
```

Comment:
The EditableTextLabel is internally wrapped in a div. Here it is nested in a p-tag which creates the following error for me: 

`Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.`

According to [stackoverflow](https://stackoverflow.com/questions/41928567/div-cannot-appear-as-a-descendant-of-p)  this happens because there should only be inline-block elements in a p-tag and div is by default a block element. 
Changing line 105 of EditableTextLabel to a span-tag with no style fixed it for me and had no visual sideeffects (as it should because a div set with inline-style should be the same as a span [link](https://stackoverflow.com/questions/1611065/span-vs-div-inline-block))

Reply:
Thanks for supplying the fix right away :)